editor.planx.uk/src/pages/FlowEditor/components/Flow/components/Tag.tsx

```diff
@@ -8,12 +8,12 @@ import { FONT_WEIGHT_SEMI_BOLD } from "theme";
 
 export const TAG_DISPLAY_VALUES: Record<
   NodeTag,
-  { color: keyof Palette["nodeTag"]; displayName: string; isEditable?: Role[] }
+  { color: keyof Palette["nodeTag"]; displayName: string; editableBy?: Role[] }
 > = {
   placeholder: {
     color: "blocking",
     displayName: "Placeholder",
-    isEditable: ["platformAdmin"], // if new roles are added, we should update the canEdit() in ComponentTagSelect.tsx
+    editableBy: ["platformAdmin"], // if new roles are added, we should update the canEdit() in ComponentTagSelect.tsx
```

Comment:
If we use the existing `state.getUserRoleForCurrentTeam()` we won't need this comment or to maintain and update `canEdit` which will share the same functionality 👌